### ISSUE TITLE
False-positive activation of 1TBS rule

### ISSUE THREAD

**Issue by petertrr** [cqfn/diKTat#215]
For example, for this code:
```kotlin
fun List<RulesConfig>.getRuleConfig(rule: Rule): RulesConfig? {
    return this.find { it.name == rule.ruleName() }
}
```
rule gives the following error:
```
[INFO]    [ktlint] C:\...\diktat-common\src\main\kotlin\org\cqfn\diktat\common\config\rules\RulesConfigReader.kt:61:51: [BRACES_BLOCK_STRUCTURE_ERROR] braces should follow 1TBS style: no newline before closing brace
```
